### PR TITLE
session 2 labs (WIP)

### PR DESCRIPTION
**Pull request by aphollis**
could you provide an example of an appropriate use of Assert in my fibonacci exercise?

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (PythonCHB) [inline .idea/vcs.xml]: you don't generally want to add this kind of thing to git.

try removing it:

``git rm .idea/vcs.xml``

now that ``.idea`` is in ``.gitignore`` it shouldn't get added accidentally. 

But I generally avoid every using ``git add *`` or git add .``, but rather, add only the files you specifcally want to add,

File: students/aphollis/session02/fizzbuzz.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+"""fizzbuzz"""
+
+for i in range(1,101):
+
+    if i % 3 == 0 and i % 7 == 0:
```
Comment (PythonCHB) [inline students/aphollis/session02/fizzbuzz.py]: it's supposed to be 3 and 5 -- but the logic is perfect.

File: students/aphollis/session02/series.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+    if len(list) == n:
+        return(list[n-1])
+    else:
+        list.append(next_up)
+        sum_series(n, list)
+
+
+fibonacci(6)
+lucas(6)
```
Comment (PythonCHB) [inline students/aphollis/session02/series.py]: So this where you can use asserts:

assert fibonacci(6) == 8

etc...

Ideally, you put them in an ``if __name__ == "__main__":`` block

```
if __name__ == "__main__":
    assert fibonacci(6) == 8
    etc...
```

**Comment by PythonCHB** [comment]
I'm not going to merge until the .idea file is removed.

File: students/aphollis/session02/fizzbuzz.py (hunk 2 above)
Comment (aphollis) [inline students/aphollis/session02/fizzbuzz.py]: fixed, no idea where 7 came from there...

**Comment by aphollis** [comment]
removed .idea

**Comment by PythonCHB** [comment]
nice job figuring out git!

**Pull request merged**